### PR TITLE
Styling table companies

### PR DESCRIPTION
**Pull request by NARCISA-LUCIA**
# Description
**Why**: Companies table needs improvements
**What**: Styling to companies table
**How**: 
Styling cursor pointer when hovering over table input
Add tooltip for remove icon
# Validation
Manual check that all functionality is working properly
**Test**:
**Existing/Added**: No tests added

### CODE REVIEW

File: src/app/company-page/dialog/remove-dialog/remove-dialog.component.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
 <p>Are you sure you want to remove company {{data.name}}?</p>
+<div class="button-click">
 <button mat-raised-button (click) = "onCancel()">Cancel</button>
```
Comment (sabaudragos) [inline src/app/company-page/dialog/remove-dialog/remove-dialog.component.html]: Could you please arange the code here? The indentation is missing.

File: src/app/company-page/dialog/remove-dialog/remove-dialog.component.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.button-click {
```
Comment (sabaudragos) [inline src/app/company-page/dialog/remove-dialog/remove-dialog.component.css]: Would `button-wrapper` be more suggestive? Or even `button-box`?

File: src/app/company-page/company-page.component.css [hunk 3]
```diff
@@ -1,3 +1,17 @@
-table{
-    width:100%;
-}
\ No newline at end of file
+table {
+    width: 1200px;
+}
+
+.demo-button {
```
Comment (sabaudragos) [inline src/app/company-page/company-page.component.css]: Would `add-data-button` be more intuitive for who reads the code? Or maybe `add-company-button`?

**Review by sabaudragos — CHANGES_REQUESTED** [review]
I've left a few comments on some minor issues, naming and code indentation. 
But the overall code looks good, you did a good job.

File: src/app/company-page/dialog/remove-dialog/remove-dialog.component.html (hunk 1 above)
Comment (NARCISA-LUCIA) [reply src/app/company-page/dialog/remove-dialog/remove-dialog.component.html]: Done

File: src/app/company-page/dialog/remove-dialog/remove-dialog.component.css (hunk 2 above)
Comment (NARCISA-LUCIA) [reply src/app/company-page/dialog/remove-dialog/remove-dialog.component.css]: I changed to `button-box-remove`

File: src/app/company-page/company-page.component.css (hunk 3 above)
Comment (NARCISA-LUCIA) [reply src/app/company-page/company-page.component.css]: Done

**Review by sabaudragos — APPROVED** [review]

**Pull request merged**